### PR TITLE
add alter after

### PR DESCRIPTION
**Pull request by vaibhavmule**

### CODE REVIEW

File: src/masonite/orm/grammar/BaseGrammar.py [hunk 1]
```diff
@@ -93,6 +94,7 @@ def _compile_alter(self):
                 if column.length
                 else "",
                 nullable="" if column.is_null else " NOT NULL",
+                after=" AFTER " + "`{0}`".format(column.after_column) if column.after_column else ""
```
Comment (vaibhavmule) [inline src/masonite/orm/grammar/BaseGrammar.py]: @josephmancuso what could be a better way to write it?

Comment (josephmancuso) [reply src/masonite/orm/grammar/BaseGrammar.py]: you have to put this string inside an `after_column_string()` method on the `MySQLGrammer` class and then change this line to 

```
after=self.after_column_string().format(after=column.after_column) if column.after_column else ""
```

On the `MySQLGrammer` class create a method like this:

```python
def after_column_string(self):
    return "AFTER {after}"
```

**Review by josephmancuso — CHANGES_REQUESTED** [review]
Awesome. Just a small change to make the code more adaptable

**Comment by josephmancuso** [comment]
**Also whenever you create a method on the MySQLGrammer class, be sure to also put the same exact method on the `GrammerExample` class so we know what methods to add to the other grammars**

**Comment by vaibhavmule** [comment]
@josephmancuso where is `GrammarExample`?

File: src/masonite/orm/grammar/BaseGrammar.py [hunk 2]
```diff
@@ -93,6 +94,8 @@ def _compile_alter(self):
                 if column.length
                 else "",
                 nullable="" if column.is_null else " NOT NULL",
+                after=self.after_column_string().format(after=column.after_column) if column.after_column else ""
```
Comment (josephmancuso) [inline src/masonite/orm/grammar/BaseGrammar.py]: actually just modify this `after=self._compile_column(column.after_column)`

File: src/masonite/orm/grammar/GrammarExample.py [hunk 3]
```diff
@@ -84,3 +84,6 @@ def column_string(self):
 
     def value_string(self):
         return "'{value}'{seperator}"
+
+    def after_column_string(self):
+        return " AFTER `{after}`"
```
Comment (josephmancuso) [inline src/masonite/orm/grammar/GrammarExample.py:89]: With the change above you can remove the back ticks here

**Review by josephmancuso — CHANGES_REQUESTED** [review]

**Comment by josephmancuso** [comment]
the reason we make this change is it will be easier to manage new grammars and also the column formatting can change. MySQL is 
```
`column`
```

but mssql is 

```
[column]
```

File: src/masonite/orm/grammar/BaseGrammar.py (hunk 2 above)
Comment (vaibhavmule) [reply src/masonite/orm/grammar/BaseGrammar.py]: That will just have ``name``

**Pull request merged**